src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/model/OffenderAliasPendingDeletion.java

```diff
@@ -45,6 +46,6 @@
     private LocalDate birthDate;
 
     @Singular
-    @OneToMany(mappedBy = "offenderAlias")
+    @OneToMany(mappedBy = "offenderAlias", fetch = FetchType.EAGER)
```

Comment:
Because the API that triggers the fetching of this object is async, JPA was losing track of the session and so couldn't lazily load this field.  Because we are transforming the entirety of this object anyway, there is no problem with eagerly loading the associated offenderBookings to resolve this issue.

Reply:
Bet that was fun working that out. :+1: